components/util/parsely.coffee

```diff
@@ -0,0 +1,20 @@
+request = require 'superagent'
+{ reject } = require 'underscore'
+
+module.exports =
+
+  topParselyArticles: (section, articleSlug, key, secret, cb) ->
```

Comment:
The move to `util` was influenced by the likelihood that 1. we'll use more parsely fetching in homepage/tag pages and 2. running the class-level `Article.topParselyArticles` in the `Articles` subapp just seemed a bit off to me.